locales/pl.json

```diff
@@ -0,0 +1,74 @@
+{
+	"select-user-type.who_you_are": "Kim jesteś? (będziesz to mógł zmienić później)",
+	"select-user-type.driver": "Kierowca",
+	"select-user-type.passenger": "Pasażer",
+	"request-phone.type_your_phone": "Proszę przesłać do nas swój numer telefonu. Nie udostępnimy go bez twojej wyraźnej zgody.",
+	"request-phone.button_text": "Wyślij numer",
+	"request-phone.all_set": "👌 Wszystko jest już ustawione, możesz już zamówić przejazd!",
+	"passenger-index.choose_taxi_type": "Jaki chcesz rodzaj taksówki?",
+	"passenger-index.car": "🚗 Samochód",
+	"passenger-index.motorbike": "🏍 Skuter / Motocykl",
+	"passenger-index.s": "🔧 Ustawienia",
+	"passenger-index.error_only_known_type": "Nieznany typ, proszę wybrać samochód, motor lub ustawienia.",
+	"passenger-request-location.provide_location": "Aby znaleźć kierowców w pobliżu, proszę udostępnić swoją lokalizację.",
+	"passenger-request-destination.provide_destination": "Gdzie chcesz jechać? Proszę wpisać dokładną lokalizację (w sposób zrozumiały dla wyszukiwarki GPS) lub opisać to miejsce.",
+	"order-submitted.order_submitted": "⏳ Szukam kierowców... Jeśli znalazłeś transport, wciśnij proszę przycisk poniżej 👇",
+	"order-submitted.cancel": "Zakończ wyszukiwanie",
+	"order-submitted.error_incorrect_input": "Nieznany wybór, proszę spróbować ponownie.",
+	"order-cancelled.order_cancelled": "👌 Dziękuję, wyszukiwanie kierowców zakończone.",
+	"driver-select-vehicle-type.select": "Wybierz typ pojazdu jaki masz zamiar używać do przewozów.",
+	"driver-select-vehicle-type.motorbike": "🏍 Skuter / Motocykl",
+	"driver-select-vehicle-type.car": "🚗 Samochód",
+	"driver-select-vehicle-type.error_only_known_type": "Nieznany typ, proszę wybrać samochód lub motor.",
+	"driver-explain-checkins.text": "Prawie skończyliśmy, oto kilka spraw do odnotowania. Telegram nie sprawdza pozycji GPS na bieżąco. Zatem zamelduj się za każdym razem gdy zmienisz swoje miejsce postoju, a gdy jesteś w trybie oczekiwania na klienta, potwierdź swoją lokalizację co godzinę. To nam pomoże znaleźć pasujących pasażerów. Kliknij \"Dalej\".",
+	"driver-explain-checkins.next": "Dalej",
+	"driver-request-location.provide_location": "Aby pasażerowie mogli cię znaleźć, proszę udostępnić swoją aktualną lokalizację.",
+	"driver-explain-whats-next.text": "Wszystko ustawione! Powiadomimy cię o pasażerach lub zamówieniach w pobliżu. Nie wyłączaj tej aplikacji i zamelduj się co godzinę. Kliknij \"Dalej\".",
+	"driver-explain-whats-next.next": "Dalej",
+	"driver-index.checkin": "📍 Zamelduj się",
+	"driver-index.mute": "🔕 Uśpij",
```

Comment:
I'd suggest "wycisz" for "mute" and "wyłącz wyciszenie" for unmute. This way you can rephrase the "driver-unmute.unmute_ok" string, so it sounds less awkward